docs/reference/commandline/swarm_init.md

```diff
@@ -27,7 +27,9 @@ Options:
       --external-ca value               Specifications of one or more certificate signing endpoints
       --force-new-cluster               Force create a new cluster from current state
       --help                            Print usage
+      --max-snapshots int               Number of additional Raft snapshots to retain
```

Comment:
What's the ordering used here? If alphabetical, then this and `--listen-addr` should be swapped